x/incentives/keeper/msg_server.go

```diff
@@ -27,21 +30,24 @@ func NewMsgServerImpl(keeper *Keeper) types.MsgServer {
 var _ types.MsgServer = msgServer{}
 
 // CreateGauge creates a gauge and sends coins to the gauge.
+// Creation fee is charged from the address and sent to the txfees module to be burned.
 // Emits create gauge event and returns the create gauge response.
 func (server msgServer) CreateGauge(goCtx context.Context, msg *types.MsgCreateGauge) (*types.MsgCreateGaugeResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
+	k := server.keeper
```

Comment:
not needed, below `server.keeper` is still referenced. Maybe best to embed the keeper into the `msgServer`

Reply:
it's for convinence